docs/users/bpmn-support.adoc

```diff
@@ -471,25 +549,25 @@ The event definition can be defined on the event or on the definitions.
 [cols="1,1,4", options="header"]
 .Gateways
```

Comment:
⚠️ I don't see icons here whereas the icon files have been added in the images folder

Reply:
Good catch 🙂

Reply:
fixed in 460ba58a75d74398e626411ecb8dc8fc7e43eefc